src/pipeline.ts

```diff
@@ -72,9 +72,9 @@ export interface GitHubWorkflowProps extends PipelineBaseProps {
    * Names of GitHub repository secrets that include AWS credentials for
    * deployment.
    *
-   * @default - `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY`.
+   * @default - Get AWS credentials from GitHub secrets `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY`.
    */
-  readonly awsCredentials?: AwsCredentialsSecrets;
+  readonly awsCredentials?: AwsCredentialsProvider;
```

Comment:
This would be a breaking change. Please let me know if we can live with that, and I'll document it in the commit message.

Reply:
Of course, I'll add it back and introduce a new property instead.